### PR TITLE
dxwnd: Add version 2.05.58

### PR DESCRIPTION
**Pull request by snowman**

### CODE REVIEW

File: bucket/dxwnd.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "homepage": "https://sourceforge.net/projects/dxwnd/",
+    "description": "Window hooker to run fullscreen programs in window and much more.",
+    "version": "2_05_58",
+    "license": "GPL-3.0-or-later",
+    "url": "https://downloads.sourceforge.net/project/dxwnd/Latest%20build/v2_05_58_build.rar",
+    "hash": "sha1:86d32d6c934c34cce383d3f8a17d2fe0dc0fc302",
+    "pre_install": [
```
Comment (Ash258) [inline bucket/dxwnd.json:8]: Do not use array for one string

File: bucket/dxwnd.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "homepage": "https://sourceforge.net/projects/dxwnd/",
+    "description": "Window hooker to run fullscreen programs in window and much more.",
+    "version": "2_05_58",
+    "license": "GPL-3.0-or-later",
```
Comment (Ash258) [inline bucket/dxwnd.json:5]: version
description
homepage
license

**Review by Ash258 — CHANGES_REQUESTED** [review]
The version does not use underscore

image

File: bucket/dxwnd.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "hash": "sha1:86d32d6c934c34cce383d3f8a17d2fe0dc0fc302",
+    "pre_install": "if (!(Test-Path \"$persist_dir\\dxwnd.ini\")) { New-Item \"$dir\\dxwnd.ini\" | Out-Null }",
+    "shortcuts": [
+        [
+            "dxwnd.exe",
+            "DxWnd"
+        ]
+    ],
+    "persist": "dxwnd.ini",
+    "checkver": "v([\\d_]+)_build.rar",
```
Comment (Ash258) [inline bucket/dxwnd.json:16]: You are matching underscore, but the versioning is using dots.

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — CHANGES_REQUESTED** [review]
What is `winmm.ini` and `tweaks.ini`?

**Comment by issaclin32** [comment]
superseded by #7629

**Pull request closed without merging**